modules/edu.gemini.seqexec.engine/src/main/scala/edu/gemini/seqexec/engine/package.scala

```diff
@@ -326,12 +386,14 @@ package object engine {
       case Poll                        => Logger.debug("Engine: Polling current state")
       case GetState(f)                 => getState(f)
       case ActionStop(id, f)           => Logger.debug("Engine: Action stop requested") *> actionStop(id, f)
+      case ActionResume(id, i, cont)   => Logger.debug("Engine: Action resume requested") *> actionResume(id, i, cont)
```

Comment:
Will these generate front side events too?

Reply:
Eventually, yes. It has to cause a refresh of the step display.